config/v1/types_infrastructure.go

```diff
@@ -354,6 +354,15 @@ type AWSPlatformSpec struct {
 	// There must be only one ServiceEndpoint for a service.
 	// +optional
 	ServiceEndpoints []AWSServiceEndpoint `json:"serviceEndpoints,omitempty"`
+
+	// ResourceTags is a list of additional tags to apply to AWS resources created for the cluster.
```

Comment:
Based on the latest version of the enhancement, this description needs to be updated, as a reminder, this is user facing documentation.

Things of note:

- Any validations that we are going to apply
  - Maximum of 5 user tags applicable
  - Tags from users must not be in the `kubernetes.io` or `openshift.io` namespaces
- Interaction with other tag definitions
  - Local kubernetes resource tag definitions have precedence over these
  - Status tags will be merged with preference to the spec tags
- Clarification on what will happen when a tag is removed from the list